ui/accordion.info/sample/main.reel/main.js

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
Sorry as a matter for fact please don't write jsdoc in here, b/c they could collide w/ other jsdoc, e.g. there could be another Main class